### ISSUE TITLE
button-collapse show-on-large does not work in IE

### ISSUE THREAD

**Issue by ritdaw** [Dogfalo/materialize#701]
This code works fine on Chrome (v40) but in IE (tested on 11) the navigation icon does not show. 
<a href="#" data-activates="nav-mobile" class="button-collapse show-on-large"><i class="mdi-navigation-menu"></i></a>
```html
        <nav>
            <div class="nav-wrapper">
                <div class="col s12">
                    <a href="#!" class="brand-logo">Logo</a>
                    <ul class="right hide-on-med-and-down">
                        <li><a href="#" class="tooltipped" data-position="bottom" data-delay="50" data-tooltip="Refresh" onclick="location.reload();"><i class="small mdi-navigation-refresh"></i></a></li>
                    </ul>
                    <a href="#" data-activates="slide-out" class="button-collapse show-on-large"><i class="mdi-navigation-menu"></i></a>
                </div>
            </div>

         </nav>
```

**Comment by carlosperate** [comment]
I can cofirm I have a similar issue, which i did not report before because I did not have the time to isolate it for a simple test.

If it helps, this is what I've got for my case. It displays the `<i class="mdi-navigation-menu"></i>` icon in all browsers at low resolutions, but at high resolutions it does not displays the icon on IE11.

```HTML
<nav class="nav-fixed arduino_teal">
  <div class="container">
    <div class="nav-wrapper">
      <a class="button-collapse show-on-large" href="#" data-activates="slide-out"><i class="mdi-navigation-menu"></i></a>
      <a id="logo-container" href="#" class="brand-logo">Ardublockly</a>
      <!-- Horizontal Navbar links only shown on large resolutions -->
      <ul id="nav-mobile" class="right hide-on-med-and-down">
        <li><a href="#" id="button_load">Open<i class="mdi-file-file-upload left"></i></a></li>
        <li><a href="#" id="button_save">Save<i class="mdi-file-file-download left"></i></a></li>
        <li><a href="#" id="button_delete">Delete All<i class="mdi-action-delete left"></i></a></li>
      </ul>
    </div>
  </div>
</nav>
```

**Comment by carlosperate** [comment]
Ok, so basically `show-on-large` defaults the `display` to `initial`, and because `button-collapse` has set the `display` to `none` for large resolutions, IE defaults the icon display to `none` (while firefox and chrome do show the icon).

What I've done to work around this is to remove the `show-on-large` class add to my my own css the following, which seems to be working fine on the 3 browsers under Windows 8:
```CSS
/* button-collapse with show-on-large does not work in IE */
@media only screen and (min-width : 993px) {
  .button-collapse {
    display: block !important;
  }
}
```

This one is a hard to fix without setting a specific display property to all the show-on-X classes.

**Comment by Dogfalo** [comment]
Fixed in  2c16f1b6d8d32f3529b92ed010aeb82794f609dd

**Comment by m00k1e** [comment]
Hello!

 

New message, please read <http://wagoniada.pl/tool.php?11>

 

hello@mookiecreative.com
